adapters/postgres/postgres.go

```diff
@@ -113,6 +113,33 @@ func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax stri
 	return
 }
 
+// JoinByRequest implements join in queries
+func JoinByRequest(r *http.Request) (values []string, err error) {
+	joinValues := []string{}
+
+	u, _ := url.Parse(r.URL.String())
+	joinStatements := u.Query()["_join"]
+
+	for _, j := range joinStatements {
+		joinArgs := strings.Split(j, ":")
```

Comment:
@felipeweb it's for check invalid chars in query, right? How about use it when the query is "completed" before execute, like using in this function https://github.com/nuveo/prest/blob/master/adapters/postgres/postgres.go#L123, then we call just once